torchvision/prototype/transforms/functional/__init__.py

```diff
@@ -63,3 +69,13 @@
 )
 from ._misc import normalize_image_tensor, gaussian_blur_image_tensor
 from ._type_conversion import decode_image_with_pil, decode_video_with_av, label_to_one_hot
+
+# What are the migration plans for public methods without new API equivalents? There are two categories:
```

Comment:
@pmeier Reminder for migration plan on these as well. Might be already on your radar as part of the Transform Classes.